build.vsh

```diff
@@ -34,7 +34,7 @@ if vls_git_hash.exit_code != 0 {
 }
 os.setenv('VLS_BUILD_COMMIT', vls_git_hash.output.trim_space(), true)
 
-cmd := 'v -gc boehm -keepc -cg -cc $cc cmd/vls -o $full_vls_exec_path'
+cmd := 'v -gc boehm -keepc -cg -showcc -cc $cc cmd/vls -o $full_vls_exec_path'
```

Comment:
`-showcc` actually can be removed, but it was helpful for me to localize c source code.